Gruntfile.js

```diff
@@ -23,7 +23,6 @@ module.exports = function(grunt) {
         },
 
         ownJsFiles: [
-            'js/marked.js',
```

Comment:
This source file was a hidden dependency. The current version from npm displays checkboxes finally.